src/app.ts

```diff
@@ -60,6 +62,12 @@ export function generateServerlessRouter(fhirConfig: FhirConfig, supportedGeneri
     const metadataRoute: MetadataRoute = new MetadataRoute(fhirVersion, configHandler);
     app.use('/metadata', metadataRoute.router);
 
+    // Export

```

Comment:
support for export should be optional similar to Bundles

Reply:
Good idea. Adding that functionality.